src/modules/Dashboard/pages/ParticipantActivityDetails/ParticipantActivityDetails.styles.tsx

```diff
@@ -0,0 +1,10 @@
+import { styled, Box } from '@mui/material';
+
+import { theme } from 'shared/styles';
+
+export const StyledContainer = styled(Box)`
```

Comment:
You could use StyledFlexTopCenter component from 'shared/styles/styledComponents'.